data/json/monsters/nether.json

```diff
@@ -1051,8 +1051,8 @@
   {
     "id": "mon_giant_appendage",
     "type": "MONSTER",
-    "name": { "str": "Giant appendage" },
-    "description": "A gigantic appendage reaching through what looks like a portal, as if this is merely a probing piece of something much larger.",
+    "name": { "str": "giant appendage" },
+    "description": "A gigantic, probing appendage the size of a truck.  It seems to be jutting through a portal, suggesting that it's merely a smaller extension of something much larger.",
```

Comment:
I like this but think it might benefit from calling out the fact that it's the size of a truck and yet taking up about as much space as the player.  I personally think some of the horror is from pointing discrepancies like that and that your mind doesn't like them.